lib/flag.dart

```diff
@@ -48,6 +250,9 @@ class Flag extends StatelessWidget {
   Widget build(BuildContext context) {
     String countryName = country.toLowerCase();
     String assetName = 'packages/flag/res/flag/' + countryName + '.svg';
+    if (!flagsCode.contains(countryName)) {
+      return replacement ?? const SizedBox.shrink();
```

Comment:
You right, but what if you call it with null ? (User might get countryCode from API)
-> replacement would be null and you'll return null instead of a widget.

Reply:
You are right. We need this check if call it with null.